src/component/association-select.js

```diff
@@ -112,6 +112,11 @@ export class AssociationSelect {
    * @return {Promise}
    */
   buildFind() {
+    // resolve if called too early
+    if (!this.repository) {
```

Comment:
If this happens, the developer did something weird. Perhaps log a warning in stead of silently resolving.